HenFwork.MapEditing/Saves/WorldSave.cs

```diff
@@ -15,6 +15,8 @@ public record WorldSave
     {
         private const char chunk_separator = '\n';
 
+        public string WorldName { get; set; }
```

Comment:
Generalnie wiemy, że to się dotyczy świata (world), bo jest w klasie **World**Save, więc niepotrzebne jest precyzować, że to nazwa świata. Wystarczy `Name`.